test/specs/test1.js

```diff
@@ -0,0 +1,33 @@
+import { expect, browser, $ } from '@wdio/globals';
+
+describe('Test 1 - Perform Login', () => {
+  it('Should log in to the "Swag Labs" page', async () => {
+    await browser.url(`https://www.saucedemo.com/`);
+    await $('#user-name').setValue('standard_user');
+    await $('#password').setValue('secret_sauce');
+    await $('#login-button').click();
+
+    const actualUrl = await browser.getUrl();
+    const expectedUrl = 'https://www.saucedemo.com/inventory.html';
+
+    expect(actualUrl).toBe(expectedUrl);
+  });
+
+  it("Should display 'Products' title", async () => {
+    const productsTitleText = await $('span[data-test="title"]').getText();
```

Comment:
Your tests are co-dependant at the moment. Lets move login logic to the before each hook, so it is reused